enhancements/etcd/cluster-etcd-operator.md

```diff
@@ -490,7 +490,34 @@ items:
 
 ### Upgrade / Downgrade Strategy
 
-TODO
+Assume that I update the above to indicate that we're using a static pod operator like the kube-apiserver.
+It rolls out one node at a time, not coordinated with the MCO in any way, without regard for PDBs.
+It does prefer updating it's own crashlooping or unready pods to bringing down working members (we already did this).
+
+#### Upgrade from 4.3 to 4.4
+
+ 1. The 4.4-etcd-staticpod moves the /etcd/kubernetes/manifests/etcd-member.yaml to a backup location before trying to start etcd.
+ 2. This causes the 4.3-machineconfigpool to go degraded because the file that it tries to maintain is gone.
+ 3. If the master node restarts using a 4.3-machineconfigpool, the old /etcd/kubernetes/manifests/etcd-member.yaml will come back.
```

Comment:
are you sure if MCD works that way. last i checked, if it sees user changed something it owns it just marks the node degraded. and doesn't so anything.